.drone.yml

```diff
@@ -73,6 +94,9 @@ build:
       - cp $BIN/vic-ui* $BIN_TEMP_DIR
       - cp $BIN/appliance.iso $BIN_TEMP_DIR
       - cp $BIN/bootstrap.iso $BIN_TEMP_DIR
+      - mkdir $BIN_TEMP_DIR/ui
+      - cp -rf ui/installer/* $BIN_TEMP_DIR/ui
+      - sleep 1
```

Comment:
I'll try replace the `cp -rf ui/installer/*...` and try building it without `sleep 1`